.github/workflows/ci.yml

```diff
@@ -131,6 +131,15 @@ jobs:
           source ./bin/activate-hermit
           export CARGO_INCREMENTAL=0
           cargo clippy --jobs 2 -- -D warnings
+
+      - name: Install Node.js Dependencies for OpenAPI Check
```

Comment:
should this move to the lint check (we can rename that) for front end that runs in parallel? (not sure if it has to wait for cargo to do its thing however)